crates/testlang/outputs/cargo/slang_testlang/build.rs

```diff
@@ -0,0 +1,18 @@
+//! This build script is only used for local development.
+//! It is removed when publishing to crates.io.
+
+#![allow(clippy::cargo_common_metadata)]
```

Comment:
All 3 `clippy::cargo_common_metadata` are not actually triggering for me. Not sure if these are needed?

Reply:
It was failing for me, not sure wy, but now it is not.

Reply:
Done